README.rst

```diff
@@ -63,15 +63,17 @@ information.
 
 Acknowledgements
 ================
-This research was supported by the EBRAINS research infrastructure, funded from the European
-Union’s Horizon 2020 Framework Programme for Research and Innovation under the Specific Grant
-Agreement No. 945539 (Human Brain Project SGA3).
+The development of this software was supported by funding to the Blue Brain Project, a research center of the École polytechnique fédérale de Lausanne (EPFL), from the Swiss government’s ETH Board of the Swiss Federal Institutes of Technology.
+
+This research was supported by the EBRAINS research infrastructure, funded from the European Union’s Horizon 2020 Framework Programme for Research and Innovation under the Specific Grant Agreement No. 945539 (Human Brain Project SGA3).
 
 License
 =======
 MorphIO is licensed under the terms of the GNU Lesser General Public License version 3. Refer to
 COPYING.LESSER and COPYING for details.
 
+COPYRIGHT 2013–2021, Blue Brain Project/EPFL
```

Comment:
`src/readers/morphologyHDF5.cpp` is (c) 2013, so that's the start, correct?

Reply:
we said it should be 
Copyright (c) 2013-2021, Blue Brain Project/EFPL

Reply:
@mgeplf Yes, 2013 is the year of the first release of the software. But for the exact working of the copyright, please use:

`Copyright (c) 2013-2021 Blue Brain Project/EPFL`

Reply:
> we said it should be

I copied it from the 'Repository Requirements' page

Reply:
see updated 'Repository Requirements' page. 

Sorry, some pages might not be up-to date. But the 'Repository Requirements' page should be up-to-date now.